src/main/java/seedu/address/model/wedding/WeddingDate.java

```diff
@@ -0,0 +1,46 @@
+package seedu.address.model.wedding;
+
+import static java.util.Objects.requireNonNull;
+
+import java.time.LocalDate;
+
+/**
+ * Class for WeddingDate
```

Comment:
javadoc comment here could be more detailed. The date format 'DD/MM/YYYY' should be emphasised.